.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "no-use-before-define": "off",
+    "max-classes-per-file": "off",
+    "no-unused-vars": "off",
+    "no-plusplus": "off",
+    "eqeqeq": "off",
+    "max-len": "off"
+  },
```

Comment:
You should not alter the linter config files. Please follow the base instruction mentioned in [here](https://github.com/microverseinc/linters-config/tree/master/javascript)